src/main/java/com/vacation_bot/repositories/VacationTotalModelRepository.java

```diff
@@ -6,8 +6,10 @@
 /**
  * Talks to {@link VacationTotalModel} collection in the database.
  */
-public interface VacationTotalRepository extends MongoRepository<VacationTotalModel, String> {
+public interface VacationTotalModelRepository extends MongoRepository<VacationTotalModel, String> {
 
     VacationTotalModel findByUserIdAndYear(String id, int year);
 
+    VacationTotalModel findByUserId(String userId);
```

Comment:
Are you using this method just in tests? It's not a good practice to create methods on production side just for test usage. User instead `findAll()` in tests and remove this method.